Makefile

```diff
@@ -12,6 +12,8 @@ SQLITE_CRATES := ./src/infra/accounts/sqlite ./src/infra/auth-rebac/sqlite ./src
 ALL_DATABASE_CRATES := $(POSTGRES_CRATES) $(MYSQL_CRATES) $(SQLITE_CRATES)
 MIGRATION_DIRS := ./migrations/mysql ./migrations/postgres ./migrations/sqlite
 
+KAMU_CONTAINER_RUNTIME_TYPE ?= docker
```

Comment:
Should we default this to `podman` to match what our `ContainerRuntime` defaults to?

Not sure why we are running most unit tests with `podman` but all the DB setup.

Suggest poking #engineering channel to get peoples' opinion.

Reply:
Check with Mac lovers. It does not matter much for Linux env.

Reply:
> It does not matter much for Linux env.

On my Fedora the difference between `podman` and `docker` is very significant, I try not to use docker at all

Reply:
As an alternative option, use the default `docker` for macOS only